velox/exec/Window.cpp

```diff
@@ -573,9 +615,15 @@ vector_size_t Window::callApplyLoop(
 
   // This function requires that the currentPartition_ is available for output.
   VELOX_DCHECK_NOT_NULL(currentPartition_);
+
+  // Used to record the processed row count in current partition. And then
+  // passed this value to clearOutputRows to clear the real row count.
+  auto processedRowCount = -1;
```

Comment:
processedRowCount is unused now. It can be removed.

Reply:
@aditi-pandit Yes. Removed.